test/unit/extension/ComposerInfoTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\generis\test\unit\extension;
+
+use oat\generis\test\TestCase;
+use oat\oatbox\extension\ComposerInfo;
+use oat\oatbox\extension\exception\ManifestException;
+
+/**
+ * Class ComposerInfoTest
+ * @package oat\oatbox\extension
+ */
+class ComposerInfoTest extends TestCase
+{
+    public function testGetComposerJson()
+    {
+        $instance = new ComposerInfo();
+        $composerJson = $instance->getComposerJson($this->getSamplesDir());
+        $this->assertArrayHasKey('require', $composerJson);
+        $this->assertEquals('oat-sa/extension-tao-lightweight', $composerJson['name']);
+        $this->assertEquals('tao-extension', $composerJson['type']);
+    }
+
+    public function testGetComposerLock()
+    {
+        $instance = new ComposerInfo();
+        $composerLock = $instance->getComposerLock($this->getSamplesDir());
+        $this->assertArrayHasKey('packages', $composerLock);
+    }
+
+    public function testGetPackageInfo()
```

Comment:
This test case fails with error:
`oat\oatbox\extension\exception\ManifestException: /var/www/html/generis/test/samples/manifests/composer.lock file not found`